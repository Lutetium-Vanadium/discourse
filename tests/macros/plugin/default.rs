fn main() {
    let q = discourse::questions![Plugin { default: todo!() }];
}
