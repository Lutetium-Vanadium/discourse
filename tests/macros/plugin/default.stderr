error: option `default` does not exist for kind `plugin`
 --> $DIR/default.rs:2:44
  |
2 |     let q = discourse::questions![Plugin { default: todo!() }];
  |                                            ^^^^^^^
